compositions/upbound-aws-provider/apigw/definition.yaml

```diff
@@ -0,0 +1,119 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
```

Comment:
License type is incorrect. I realize we need to change headers for the existing ones as well but we should specify Apache 2.0 for new files. Sorry for the confusions.

Reply:
Fixed